test/integration/apiserver/apply/reset_fields_test.go

```diff
@@ -298,3 +297,31 @@ func TestApplyResetFields(t *testing.T) {
 		}
 	}
 }
+
+func expectConflict(t *testing.T, objRet *unstructured.Unstructured, err error, dynamicClient dynamic.Interface, resource schema.GroupVersionResource, namespace, name, where string) {
```

Comment:
That's a lot of parameters, maybe return an error so you can remove both `t` and `where`.

Reply:
Revised to do that.  PTAL.